modules/oe_webtools_laco_widget/oe_webtools_laco_widget.links.menu.yml

```diff
@@ -0,0 +1,5 @@
+oe_webtools_laco_widget.settings:
+  title: Webtools Laco Widget
+  description: 'Configure Laco Widget settings.'
+  route_name: oe_webtools_laco_widget.settings
+  parent: system.admin_config_regional
```

Comment:
This should go into system.admin_config_system

Reply:
For me it makes sense to be in "Regional and language" section because it's a language related setting.

I've discussed where to place the entry with @brummbar and @upchuk and @voidtek . This place I'm using was the most consensual. So, please, can you give some arguments to your preference?